Assets/Scripts/UI/PauseScrn/Navigation.cs

```diff
@@ -54,7 +55,7 @@ void Update()
                 ResumeSelected();
                 OptionsDeSelected();
                 NotebookDeSelected();
-                if (Input.GetKeyDown(KeyCode.Return))
+                if (Input.GetButtonDown("Interact"))
                 {
```

Comment:
Have you tested this out in scenes where you have dialogues. Pause when a dialogue is open and then navigate through the options..the z will move to the next dialogue. Thats why I did not implement it